octokit/client.py

```diff
@@ -9,11 +9,13 @@
 
 # https://code.google.com/p/uri-templates/wiki/Implementations
 
-from .resources import Resource
+from resources import Resource
```

Comment:
For me, the '.' gives me "System Error: Parent module '' not loaded, cannot perform relative import"
Also, we don't need the '.' right? Because it's in the same directory